app/src/main/java/org/apache/taverna/mobile/ui/workflowdetail/WorkflowDetailPresenter.java

```diff
@@ -192,11 +192,12 @@ public void onNext(Boolean b) {
                 }));
     }
 
+
     private Map<String, String> getDetailQueryOptions() {
 
         Map<String, String> option = new HashMap<>();
         option.put("elements", "id,title,type,uploader,preview,created-at,svg,updated-at," +
-                "description,license-type,tags");
+                "description,license-type,tags,content-uri");
```

Comment:
Looks nice, but writing hard-coded strings, make the difficult to other contributors, So please put these 
id,title,type,uploader,preview,created-at,svg,updated-at string in Constant.java